### PR TITLE
Sync to AngularDart 4.0.0-alpha

### PR DESCRIPTION
**Pull request by matanlurey**
Updates to the CHANGELOG and pubspec.yaml to follow.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 4.0.0
+
+### Breaking changes
+
+* Returning `false` from an event handler will no longer cancel the event. See
```
Comment (kevmoo) [inline CHANGELOG.md]: Is removal of the other compiler a breaking change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## 4.0.0
```
Comment (kevmoo) [inline CHANGELOG.md]: Should update the pubspec to be 4.0.0-dev

**Comment by kevmoo** [comment]
Just read your note on pubspec/changelog. Let me know...

File: CHANGELOG.md (hunk 2 above)
Comment (matanlurey) [inline CHANGELOG.md]: Will be updated shortly.

File: CHANGELOG.md (hunk 1 above)
Comment (matanlurey) [inline CHANGELOG.md]: Will be updated shortly.

**Comment by matanlurey** [comment]
Will fix upstream. Expect another sync.

**Pull request merged**